validator/client/beacon-api/json_rest_handler.go

```diff
@@ -16,13 +16,25 @@ import (
 type JsonRestHandler interface {
 	Get(ctx context.Context, endpoint string, resp interface{}) error
 	Post(ctx context.Context, endpoint string, headers map[string]string, data *bytes.Buffer, resp interface{}) error
+	GetHttpClient() *http.Client
+	GetHost() string
```

Comment:
Avoid using the `Get` prefix for getters. Simply call these `Host()` or `HttpClient()`